requirements-gpu.txt

```diff
@@ -0,0 +1,25 @@
+jax>=0.4.23
```

Comment:
I don't think we want to trust folks to update the different lists. Could you do this and in setup.sh if GPU also install the GPU requirements.txt?
https://stackoverflow.com/questions/36744143/how-can-i-define-multiple-requirement-files